_redirects

```diff
@@ -56,28 +56,14 @@
 
 /docs/api/     /docs/concepts/overview/kubernetes-api/ 301
 
-/docs/api-reference/apps/v1alpha1/definitions/     https://v1-4.docs.kubernetes.io/docs/api-reference/apps/v1alpha1/definitions/ 301
-/docs/api-reference/apps/v1beta1/operations/     https://v1-4.docs.kubernetes.io/docs/api-reference/apps/v1beta1/operations/ 301
-/docs/api-reference/authorization.k8s.io/v1beta1/definitions/     https://v1-4.docs.kubernetes.io/docs/api-reference/authorization.k8s.io/v1beta1/definitions/ 301
-/docs/api-reference/authorization.k8s.io/v1beta1/operations/     https://v1-4.docs.kubernetes.io/docs/api-reference/authorization.k8s.io/v1beta1/operations/ 301
-/docs/api-reference/autoscaling/v1/operations/     https://v1-4.docs.kubernetes.io/docs/api-reference/autoscaling/v1/operations/ 301
-/docs/api-reference/batch/v1/operations/     https://v1-4.docs.kubernetes.io/docs/api-reference/batch/v1/operations/ 301
-/docs/api-reference/batch/v2alpha1/definitions/     https://v1-4.docs.kubernetes.io/docs/api-reference/batch/v2alpha1/definitions/ 301
-/docs/api-reference/certificates.k8s.io/v1alpha1/definitions/     https://v1-4.docs.kubernetes.io/docs/api-reference/certificates.k8s.io/v1alpha1/definitions/ 301
-/docs/api-reference/certificates/v1alpha1/operations/     https://v1-4.docs.kubernetes.io/docs/api-reference/certificates/v1alpha1/operations/ 301
-/docs/api-reference/extensions/v1beta1/operations/     https://v1-4.docs.kubernetes.io/docs/api-reference/extensions/v1beta1/operations/ 301
 /docs/api-reference/labels-annotations-taints/     /docs/reference/labels-annotations-taints/ 301
-/docs/api-reference/policy/v1alpha1/definitions/     https://v1-4.docs.kubernetes.io/docs/api-reference/policy/v1alpha1/definitions/ 301
-/docs/api-reference/policy/v1beta1/definitions/     https://v1-4.docs.kubernetes.io/docs/api-reference/policy/v1beta1/definitions/ 301
-/docs/api-reference/README/     https://v1-4.docs.kubernetes.io/docs/api-reference/README/ 301
-/docs/api-reference/storage.k8s.io/v1beta1/operations/     https://v1-4.docs.kubernetes.io/docs/api-reference/storage.k8s.io/v1beta1/operations/ 301
 /docs/api-reference/v1.6/*     https://v1-6.docs.kubernetes.io/docs/reference/ 301
 /docs/api-reference/v1.7/*     https://v1-7.docs.kubernetes.io/docs/reference/ 301
 /docs/api-reference/v1.8/*     https://v1-8.docs.kubernetes.io/docs/api-reference/v1.8/:splat 301
 /docs/api-reference/v1.9/     /docs/reference/generated/kubernetes-api/v1.9/ 301
-/docs/api-reference/v1/definitions/     /docs/api-reference/v1.9/ 301
-/docs/api-reference/v1/operations/     /docs/api-reference/v1.9/ 301
-/docs/api-reference/v1.9/      /docs/reference/generated/kubernetes-api/v1.9/ 301
+/docs/api-reference/v1/definitions/    /docs/reference/generated/kubernetes-api/v1.10/ 301
+/docs/api-reference/v1/definitions.html /docs/reference/generated/kubernetes-api/v1.10/ 301
```

Comment:
line 65 was added because it is still used in the generated kubectl docs. We may consider fix it in the upstream later.